src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -190,24 +196,29 @@ public void feederMain() {
         } else if (oi.reverseIntake()) {
             startFeeder(-feederSpeed);
         } else {
+            gathererState = gathererUp;
             holdFeeder();
         }
     }
 
     public void startFeeder(double speed) {
         RESET_ENCODER = true;
         feederPid.setReference(speed, ControlType.kDutyCycle);
+        gathererState = gathererDown;
```

Comment:
instead of changing gatherer states here, which makes the state tracking really difficult, you might consider just starting the intake.

Reply:
fixed